internal/notifier/gitea.go

```diff
@@ -76,6 +76,14 @@ func NewGitea(commitStatus string, addr string, token string, certPool *x509.Cer
 		}
 	}
 
+	if proxyURL != "" {
+		parsedProxyURL, err := url.Parse(proxyURL)
+		if err != nil {
+			return nil, fmt.Errorf("unable to parse proxy URL '%s', error: %w", proxyURL, err)
```

Comment:
```suggestion
			return nil, errors.New("invalid proxy URL")
```
We don't want to log the proxy as it can contain user/pass in the URL.